analyzer.js

```diff
@@ -46,7 +46,11 @@ const getAllFiles = (dir, extn, files, result, regex) => {
       continue;
     }
     const file = join(dir, files[i]);
-    if (statSync(file).isDirectory()) {
+    const fileStat = lstatSync(file);
```

Comment:
Interesting! Could you fix other places where statSync is used?

Reply:
Thanks for the detailed explanation. We can refer to this PR if we encounter similar loops in docker and other places.